spec/features/applications/show_spec.rb

```diff
@@ -150,4 +151,82 @@
       end
     end
   end
+
+  describe 'Submit an Application & No Pets on an Application' do
+    describe "when one or more pets have been added to an application and its status is 'In Progress' " do
+      it 'if condition is true: has a section to submit my application, which has a test box user to add why they are a worthy candidate' do
+        visit "/applications/#{@app4.id}"
+    
+        expect(@app4.pets.present?).to eq false
+        expect(@app4.status).to eq('In Progress')
+        expect(page).to_not have_field(:bid_reason)
+        expect(page).to_not have_button('Submit Application')
+
+        visit "/applications/#{@app2.id}"
+        
+        expect(@app2.pets.present?).to eq true
+        expect(@app2.status).to eq('Pending')
+        expect(page).to_not have_field(:bid_reason)
+        expect(page).to_not have_button('Submit Application')
+
+        visit "/applications/#{@app3.id}"
+      
+        expect(@app3.pets.present?).to eq true
+        expect(@app3.status).to eq('In Progress')
+        expect(page).to have_field(:bid_reason)
+        expect(page).to have_button('Submit Application')
+      end
+
+      describe 'when user completes the worthy candidate and clicks the submit button' do 
```

Comment:
Consider rephrasing 'worthy candidate', could be more clear that this is referring to the bid_reason field.